packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -639,6 +667,14 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     ISummarizerInternalsProvider {
     public get IContainerRuntime() { return this; }
     public get IFluidRouter() { return this; }
+    private static buildSummaryConfiguration(runtimeOptions: IContainerRuntimeOptions): ISummaryConfiguration {
+        return {
+            // the defaults
+            ...DefaultSummaryConfiguration,
+            // the runtime configuration overrides
+            ...runtimeOptions.summaryOptions?.summaryConfigOverrides,
+        };
+    }
```

Comment:
I believe `load()` is the only caller of the classes private ctor, so we should only need to mix the default summary config & the runtime overrides once.

Note that line 707 in `load()` is already providing defaults for other values while destructuring runtimeOptions.  The destructured fields are then recombined as runtimeOptions when passing to the ctor on line 802.